kolibri/plugins/user/assets/src/views/SignInPage/index.vue

```diff
@@ -541,20 +530,21 @@
       },
     },
     $trs: {
+      // TODO: Remove the comments in $trs, run the linter, fix the issues
+      // Disabling this altogether for now because we use some with crossComponentTranslator
+      /* eslint-disable kolibri/vue-no-unused-translations */
       changeLabel: {
         message: 'Change',
         context:
           '(verb) Link to change the facility to sign in when the device has more than one facility',
       },
+      signInToFacilityLabel: "Sign into '{facility}'",
+      greetUser: 'Hi, {user}',
       signInError: 'Incorrect username or password',
-      signInToFacilityLabel: "Sign in to '{facility}'",
       requiredForCoachesAdmins: 'Password is required for coaches and admins',
       documentTitle: 'User Sign In',
-      greetUser: 'Hi, {user}',
       needToMakeNewPasswordLabel: 'Hi, {user}. You need to set a new password for your account.',
```

Comment:
This was the only conflict in the merge @nucleogenesis - just confirming that it was resolved correctly? I think it was just a reorder? Thanks!